string configFile
---
int32[] lengths
geometry_msgs/Pose2D[] poses
